item.rb

```diff
@@ -0,0 +1,42 @@
+require 'time'
+
+class Item
+  attr_accessor :publish_date, :archieved, :id, :genre, :author, :source, :label
+
+  def initialize(publish_date)
+    @id = Random.rand(1..1000)
+    @publish_date = publish_date
+    @archieved = false
+  end
+
+  def add_genre=(genre)
+    @genre = genre
+    genre.add_item(self) unless genre.albums.include?(self)
+  end
+
+  def add_author(author)
+    @author = author
+    # author.add_item(self) unless author.items.include?(self)
```

Comment:
- [ ] I think you'll equally need this line of code and modify it to look something like this:

`author.add_item.push(self) unless author.add_item.include?(self)`